src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/model/ReportMetaData.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.rtd.ms.rtdmsdecrypter.model;
+
+import java.time.LocalDate;
+import java.util.HashSet;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ReportMetaData {
+  private HashSet<String> merchantList;
+  private long numCanceledTrx;
```

Comment:
Shouldn't this be `numNegativeTrx`?
Same for `totalAmountNegativeTrx`.

Reply:
I put canceled because the meaning is more appropriate in my opinion, they represent a canceled transaction. But if we decide that negative is more appropriate we can change! :)